src/_C001.ino

```diff
@@ -111,7 +111,7 @@ bool CPlugin_001(CPlugin::Function function, struct EventStruct *event, String&
           url += mapVccToDomoticz();
             # endif // if FEATURE_ADC_VCC
 
-          success = C001_DelayHandler->addToQueue(C001_queue_element(event->ControllerIndex, event->TaskIndex, url));
+          success = C001_DelayHandler->addToQueue(std::move(C001_queue_element(event->ControllerIndex, event->TaskIndex, std::move(url))));
```

Comment:
Yep, is probably a left-over as I did this first, then realized it still might use the copy constructor and then I added a move constructor to make sure copy constructor is never used.
I will check if I can remove this `std::move` wrapper and see how the compiler thinks of it.

Reply:
cppreference also describes these as prvalues:
https://en.cppreference.com/w/cpp/language/value_category#C.2B.2B11
aka a temporary, which I forgot a word for

also tangentially related, rvalue stops being rvalue in the method / function block itself:
https://godbolt.org/z/3EPcPhePf
so copy might accidentally happen after accepting && and calling something else expecting &&, but not using `std::move(value)` / `static_cast<T&&>(value)`

Reply:
Yep, so that's why the copy constructor or assignment operator should be declared with `= delete` where possible to prevent these to make sure the compiler will catch them.
Still tricky on objects that have all declared like `String`.

But as you may have seen, I have added a lot of those compiler checks in the code to help finding bugs (and now performance hits) as soon as possible.
So I'm happy to have learned about the `=delete` and `=default`, which may help a lot.